functions.txt

```diff
@@ -1,6 +1,11 @@
 <?php
 
+// placeholder constants
+define('TODO', -1);
+define('TODO_OVERLOAD', -1);
```

Comment:
Why do we need these constants?

Reply:
There are places in `functions.txt` where we use them. And their use there seems to be necessary in order to organize a kind of overloading for the function.

Reply:
> Why do we need these constants?

For example:

```php
function mt_rand ($l ::: int = TODO_OVERLOAD, $r ::: int = TODO_OVERLOAD) ::: int;
```

Reply:
ok, nice